inference-engine/tests/functional/plugin/shared/src/single_layer_tests/convolution.cpp

```diff
@@ -78,4 +78,9 @@ void ConvolutionLayerTest::SetUp() {
 TEST_P(ConvolutionLayerTest, CompareWithRefs) {
     Run();
 }
+
+TEST_P(ConvolutionLayerTest, Serialization) {
```

Comment:
I suppose, it should be `inference_engine` functional tests, because they don't use plugin functionality. We can create `ngraph_serialize` subfolder in `inference_engine` functional tests.